packages/calcite-components/src/components/chip/chip.scss

```diff
@@ -71,97 +60,111 @@
   --calcite-internal-chip-container-size: 44px;
 }
 
-:host([appearance="outline-fill"]) {
-  --calcite-chip-background-color: var(--calcite-color-foreground-1);
+.container {
+  @apply inline-flex
+  h-full
+  max-w-full
```

Comment:
h-full and max-w-full can be changed to 
`block-size: var(--calcite-container-size-content-fluid)`
`max-inline-size: var(--calcite-container-size-content-fluid)`

Reply:
Updated